### ISSUE TITLE
SystemApps Folder Issue

### ISSUE THREAD

**Issue by KedarWolf** [DrEmpiricism/Optimize-Offline#92]
I noticed the folders for the apps I removed are still there. 

Is it possible to completely delete the removed apps folders with the script?

![Untitled](https://user-images.githubusercontent.com/7151550/60075143-a706bf00-96f2-11e9-8dbb-da7299328c3f.png)

**Comment by Hyperyia** [comment]
Optimize-Offline doesn't remove appx files afaik, it just removes registry keys to prevent them from being provisioned/registered - it's generally considered to be a clean, proper and safer approach towards removing UWP/Modern appx than to remove the files, because removing them can break up sfc integrity or cause other issues.

![image](https://user-images.githubusercontent.com/32967354/60116553-8d0fbf80-9778-11e9-9e7f-07bdc822adce.png)

**Comment by KedarWolf** [comment]
Thank you. I understand now.

**Comment by DrEmpiricism** [comment]
Per the Readme:

> System Applications are a lot like Provisioned Application Packages (Metro Apps) in respect that they are provisioned and installed during the setup of Windows. During the Windows Setup component pass, setup looks for these System Applications in the default registry and provisions them for installation only if their entries are present. By removing these entries, Windows Setup does not provision them for installation.

This method is safer than force removing the System Application using its component package because it retains the default file structure. Furthermore, the force removal of System Applications' component packages can trip the dreaded "STATUS_SXS_COMPONENT_STORE_CORRUPT" flag. This is a critical component store corruption flag that will then be detected by any servicing command and Windows Update and prevent both the servicing and updating of the Operating System. The only way to remedy and fix this error is to re-install or reset the Operating System.

In short, Optimize-Offline removes the registry key the Windows Setup pass looks for when provisioning system applications for installation and integration. If the key is not present, that system application is not provisioned for installation. Moreover, the actual file structure of the system application is left in-tact within the WIM as this ensures no corruption is detected silently by Windows and then flagged as critical.

Conversely you can manually remove those directories if you so choose, but the amount of space they take up is minute and well worth keeping to ensure your system remains healthy.
